src/status_im/contacts/screen.cljs

```diff
@@ -131,7 +133,7 @@
         (when @show-toolbar-shadow?
           [linear-gradient {:style  st/contact-group-header-gradient-bottom
                             :colors st/contact-group-header-gradient-bottom-colors}])]
-       (if (pos? (+ @dapps-count @people-count))
+       (if (pos? (+ @dapps-count @state-of-the-dapps-count @people-count))
```

Comment:
This function (`contact-list`) should be defined with `defview`. It will make the code cleaner  by removing all these dereferences from here.